pandas/core/arrays/categorical.py

```diff
@@ -379,6 +382,137 @@ def __init__(
         self._dtype = self._dtype.update_dtype(dtype)
         self._codes = coerce_indexer_dtype(codes, dtype.categories)
 
+    @classmethod
```

Comment:
i agree with the others

either remove this or make it identical to get_dummies

Reply:
I think the discussion around this was more about the necessity/ implementation of `to_dummies` - there currently isn't an equivalent of `from_dummies` in pandas, which is what originally precipitated the issue. But note taken re. `to_dummies` if that was the intention.